### PR TITLE
Implement tab completion for tags.

### PR DESCRIPTION
**Pull request by jsenich**
Here's my initial attempt at implementing tab completion for tags (Issue #108).  It doesn't yet recognize trying to complete using "tags/" like described in the issue, but cycles through a combined list of branches and tags similar to how this is handled in the bash prompt implementation.

### CODE REVIEW

File: GitTabExpansion.ps1 [hunk 1]
```diff
@@ -74,6 +74,12 @@ function script:gitBranches($filter, $includeHEAD = $false) {
         foreach { $prefix + $_ }
 }
 
+function script:gitTags($filter) {
+    git --git-dir="$(Get-GitDirectory)" for-each-ref --format='%(refname:short)' refs/tags |
```
Comment (dahlbyk) [inline GitTabExpansion.ps1]: As we celebrate the anniversary of this comment... do you want to take a crack at updating this or should I?

**Comment by jsenich** [comment]
Modified my commit to use the simpler git tag rather than manually reading the refs.

File: GitTabExpansion.ps1 [hunk 2]
```diff
@@ -74,6 +74,12 @@ function script:gitBranches($filter, $includeHEAD = $false) {
         foreach { $prefix + $_ }
 }
 
+function script:gitTags($filter) {
+    git tag |
+        where { $_ -like "$filter*" } |
+        foreach { $_ }
```
Comment (JayBazuzi) [inline GitTabExpansion.ps1]: What is this line for?

Comment (jsenich) [inline GitTabExpansion.ps1]: It provides tab completion based on tag names and is called from GitTabExpansion().

Comment (JayBazuzi) [inline GitTabExpansion.ps1]: I'm asking specifically about the line `foreach { $_ }`.

Try this: remove that line, try TAB-completion on tags, and see if it still works.

Comment (jsenich) [inline GitTabExpansion.ps1]: Good call. That line wasn't  necessary.

**Comment by dahlbyk** [comment]
Thank you!

For future reference, watchers won't get a notification on `push` to a PR, so you may want to bump a thread to which you've pushed a fix. :sparkles:

**Pull request merged**